README.md

```diff
@@ -1,117 +1,36 @@
-
-    4. *Granular Threat Detection*: Sigma rules can be fine-tuned to detect specific behaviors or patterns associated with an outbreak. Whether identifying known indicators of compromise (IoCs) or recognizing unusual network traffic patterns, Sigma rules allow for granular and targeted threat detection, enhancing the ability to detect outbreaks early.
-
-    5. *Adaptability to New Threats*: Outbreak alerts often involve new and evolving threats. You can modify and adapt Sigma rules to address the characteristics of emerging threats. This adaptability is crucial in staying ahead of cyber criminals who constantly enhance their tactics, techniques, and procedures (TTPs).
-
-3. **YARA Rules**: YARA rules help detect outbreaks and other security threats. It is a pattern-matching and rule-based language designed to help researchers and defenders identify and classify malware based on textual or binary patterns. Here's how YARA rules can be beneficial in detecting outbreak alerts:
-
-    1. *Pattern Matching*: YARA rules allow for the definition of specific patterns or signatures associated with malicious files or behaviors. In the context of outbreak alerts, these patterns could include unique strings, file structures, or other characteristics that indicate a particular malware or threat.
-
-    2. *Flexibility and Customization*: YARA rules are highly flexible and can be customized to target specific threat attributes. Security researchers can create rule sets that focus on the unique features of an outbreak, allowing for precise detection tailored to the distinct characteristics of the threat.
-
-    3. *Scalability*: YARA rules can be scaled to cover a range of files and activities. This scalability is crucial in case of outbreaks where the volume of malicious activity is significant. You can apply YARA rules to large datasets for identifying and isolating hostile entities.
 
-    4. *Indicators of Compromise (IoCs)*: You can use YARA rules to define IoCs associated with an outbreak. These could include file hashes, registry entries, or network behavior patterns. By incorporating these IoCs into YARA rules, organizations can automate the detection of malicious entities based on known indicators.
+## Supercharge Your Threat Hunting
 
-    5. *Community Collaboration*: YARA rules benefit from community collaboration, similar to Sigma rules. Security professionals often share YARA rules to help others detect and respond to emerging threats. This collaborative approach enables a broader and more collective defense against outbreaks.
+Dive deep into the details of every Outbreak Alert with our robust Threat Hunt Rules. Whether through the centralized power of Fortinet Fabric, the adaptability of Sigma, or the pattern-matching precision of YARA, your organization will be equipped to detect and neutralize threats like never before.
```

Comment:
Create a new page for threat hunt rules and add all removed in it and link it here

Reply:
Removed info is useful info the user can be retained.

Reply:
@cs-abhishek-shukla  this is still pending

Reply:
The removed info is now a part of a new file [Overview of Outbreak Response Framework](https://github.com/cs-abhishek-shukla/solution-pack-outbreak-response-framework/blob/release/2.0.0/docs/background-information.md)